tensorflow/tools/dist_test/python/mnist_replica.py

```diff
@@ -52,29 +52,24 @@
 flags.DEFINE_boolean("download_only", False,
                      "Only perform downloading of data; Do not proceed to "
                      "session preparation, model definition or training")
-flags.DEFINE_integer("worker_index", 0,
-                     "Worker task index, should be >= 0. worker_index=0 is "
+flags.DEFINE_integer("task_index", 0,
+                     "Worker task index, should be >= 0. task_index=0 is "
                      "the master worker task the performs the variable "
                      "initialization ")
 flags.DEFINE_integer("num_workers", 2,
                      "Total number of workers (must be >= 1)")
-flags.DEFINE_integer("num_parameter_servers", 1,
-                     "Total number of parameter servers (must be >= 1)")
+flags.DEFINE_integer("num_gpu", 4,
+                     "Total number of gpus for each machine (must be >= 1")
```

Comment:
Can you set the default here to be 1, and make it so that 0 is also a valid value? (You'll need to change the code for constructing the worker device, so that it uses `"/cpu:0"` when `FLAGS.num_gpus` is 0.)

Also, can you rename this flag from `num_gpu` to `num_gpus` for consistency with the other flags?